facets.wdl

```diff
@@ -5,8 +5,8 @@ workflow facets_workflow {
     File normal_bam
     File normal_bam_index
 
-    File? pileup_snp_vcf = "gs://fc-f36b3dc8-85f7-4d7f-bc99-a4610229d66a/broadinstitute/reference/hg19/db/dbsnp/dbsnp_134_b37.leftAligned.vcf"
-    File? pileup_snp_vcf_index = "gs://fc-f36b3dc8-85f7-4d7f-bc99-a4610229d66a/broadinstitute/reference/hg19/db/dbsnp/dbsnp_134_b37.leftAligned.vcf.idx"
+    File? pileup_snp_vcf = "gs://fc-f36b3dc8-85f7-4d7f-bc99-a4610229d66a/common_sites/00-common_all.vcf.gz"
```

Comment:
I think an inline comment is warranted explaining what database of variants this is

Reply:
Added: https://github.com/vanallenlab/facets/commit/fe88a4afcee38f1faba55a16f74c8fe3b3df4748

Reply:
Looks like you removed the preset defaults. Are we still keeping around a mirror of the dbSNP sites somewhere in GCS that we can provide in a default config or something?

Reply:
Yup, that will be in a default config on firecloud for our group